src/server/game/Handlers/ChatHandler.cpp

```diff
@@ -194,10 +194,10 @@ void WorldSession::HandleMessagechatOpcode(WorldPacket& recvData)
                 return;
         }
     }
-    // LANG_ADDON should not be changed nor be affected by flood control
+
```

Comment:
delete this line

Reply:
[ff0dfea](https://github.com/azerothcore/azerothcore-wotlk/pull/12603/commits/ff0dfea46d040e9e04ef7cbb13d0eb25e34d0bed)